birdhouse/config/portainer/config/proxy/conf.extra-service.d/portainer.conf.template

```diff
@@ -1,5 +1,5 @@
     location /portainer/ {
-        proxy_pass http://${PAVICS_FQDN}:9000/;
+        proxy_pass http://portainer:9000/;
```

Comment:
Bypasses Magpie/Twitcher auth.

Reply:
Yes.... portainer does manage its own authorization so this is not necessarily insecure but we could make it more secure by putting it behind twitcher as well.  I'd be open to double-protecting portainer since its such a powerful tool.

(also, I didn't make any changes to which services are behind twitcher in this PR, see my comment above)

Reply:
To Do: https://github.com/bird-house/birdhouse-deploy/issues/335